Cannot build with HHVM 3.9.0

It appears the `IniSetting` class has changed between HHVM 3.8.1 and HHVM 3.9.0.

When I run `./build.sh`, it fails with the following:

```
make
Scanning dependencies of target mongo
[ 33%] Building CXX object CMakeFiles/mongo.dir/src/bson.cpp.o
/tmp/builds/mongofill-hhvm/src/bson.cpp: In member function ‘virtual void HPHP::mongoExtension::moduleLoad(const Map&, HPHP::Hdf)’:
/tmp/builds/mongofill-hhvm/src/bson.cpp:78:24: error: ‘const Map’ has no member named ‘get_ptr’
     auto ini_val = ini.get_ptr("apc.enable");
                        ^
make[2]: *** [CMakeFiles/mongo.dir/src/bson.cpp.o] Error 1
make[1]: *** [CMakeFiles/mongo.dir/all] Error 2
make: *** [all] Error 2
```

--- a/src/bson.cpp
+++ b/src/bson.cpp
@@ -75,7 +75,7 @@ class mongoExtension : public Extension {
   
   virtual void moduleLoad(const IniSetting::Map& ini, Hdf hdf) {
     EnableApc = true;
-    auto ini_val = ini.get_ptr("apc.enable");
+    auto ini_val = Config::Get(ini, hdf, "apc.enable");
     if (ini_val != nullptr) {
       ini_on_update(*ini_val, EnableApc);
     }

--- a/src/decode.cpp
+++ b/src/decode.cpp
@@ -157,7 +157,7 @@ static void bsonToObject(
 ) {
   output->add(
     String(bson_iter_key(iter)),
-    instanceNewObject(className, params)
+    Variant(instanceNewObject(className, params))
   );
 }
 

--- a/src/encode.cpp
+++ b/src/encode.cpp
@@ -149,12 +149,12 @@ void mongoCodeToBSON(const Object& value, const char* key, bson_t* bson) {
   bson_t child;
   bson_init(&child);
   fillBSONWithArray(
-    value->o_get("scope", true, s_MongoCode.get()).toArray(),
+    value->o_get("scope", true, String(s_MongoCode.get())).toArray(),
     &child 
   );
   
   bson_append_code_with_scope(bson, key, -1,
-    value->o_get("code", true, s_MongoCode.get()).toString().c_str(),
+    value->o_get("code", true, String(s_MongoCode.get())).toString().c_str(),
     &child
   );
 }

